diff --git a/lisp/autorevert.el b/lisp/autorevert.el
index 97a122b7bc..940796bb94 100644
--- a/lisp/autorevert.el
+++ b/lisp/autorevert.el
@@ -508,6 +508,7 @@ suppressed by setting `auto-revert-verbose' to nil.
 This function calls the hook `global-auto-revert-mode-hook'.
 It displays the text that `global-auto-revert-mode-text'
 specifies in the mode line."
+  :init-value t
   :global t :group 'auto-revert :lighter global-auto-revert-mode-text
   (auto-revert-set-timer)
   (if global-auto-revert-mode
diff --git a/lisp/cus-start.el b/lisp/cus-start.el
index 62a4f634b5..85d2ae7e2f 100644
--- a/lisp/cus-start.el
+++ b/lisp/cus-start.el
@@ -152,7 +152,7 @@ Leaving \"Default\" unchecked is equivalent with specifying a default of
 	     (scroll-down-aggressively windows
 				       (choice (const :tag "off" nil) float)
 				       "21.1")
-	     (line-spacing display (choice (const :tag "none" nil) number)
+	     (line-spacing display (choice (const :tag "0.1" 0.1) float)
 			   "22.1")
 	     (cursor-in-non-selected-windows
 	      cursor ,cursor-type-types nil
diff --git a/lisp/delsel.el b/lisp/delsel.el
index f5fe7cf793..84b864e725 100644
--- a/lisp/delsel.el
+++ b/lisp/delsel.el
@@ -77,6 +77,7 @@ point regardless of any selection.
 
 See `delete-selection-helper' and `delete-selection-pre-hook' for
 information on adapting behavior of commands in Delete Selection mode."
+  :init-value t
   :global t :group 'editing-basics
   (if (not delete-selection-mode)
       (remove-hook 'pre-command-hook 'delete-selection-pre-hook)
diff --git a/lisp/elec-pair.el b/lisp/elec-pair.el
index 964d21f11c..49cec21f7d 100644
--- a/lisp/elec-pair.el
+++ b/lisp/elec-pair.el
@@ -635,6 +635,7 @@ If the region is active, the parentheses (brackets, etc.) are
 inserted around the region instead.
 
 To toggle the mode in a single buffer, use `electric-pair-local-mode'."
+  :init-value t
   :global t :group 'electricity
   (if electric-pair-mode
       (progn
diff --git a/lisp/files.el b/lisp/files.el
index 5fa38e46af..bd67536d53 100644
--- a/lisp/files.el
+++ b/lisp/files.el
@@ -68,7 +68,7 @@ a regexp matching the name it is linked to."
   :group 'abbrev
   :group 'find-file)
 
-(defcustom make-backup-files t
+(defcustom make-backup-files nil
   "Non-nil means make a backup of a file the first time it is saved.
 This can be done by renaming the file or by copying.
 
@@ -316,7 +316,7 @@ Includes the new backup.  Must be greater than 0."
   :group 'backup)
 (put 'kept-new-versions 'safe-local-variable 'integerp)
 
-(defcustom require-final-newline nil
+(defcustom require-final-newline t
   "Whether to add a newline automatically at the end of the file.
 
 A value of t means do this only when the file is about to be saved.
@@ -430,6 +430,7 @@ You can also set the buffer-local value of the variable
 value of this variable is nil is ignored for the purpose of
 `auto-save-visited-mode', even if `auto-save-visited-mode' is
 enabled."
+  :init-value t
   :group 'auto-save
   :global t
   (when auto-save--timer (cancel-timer auto-save--timer))
@@ -675,7 +676,7 @@ using \\[read-only-mode]."
 Maximum length of the history list is determined by the value
 of `history-length', which see.")
 
-(defvar save-silently nil
+(defvar save-silently t
   "If non-nil, avoid messages when saving files.
 Error-related messages will still be printed, but all other
 messages will not.")
diff --git a/lisp/net/goto-addr.el b/lisp/net/goto-addr.el
index 99ed14ca8b..5aa4b0dc60 100644
--- a/lisp/net/goto-addr.el
+++ b/lisp/net/goto-addr.el
@@ -296,6 +296,7 @@ Also fontifies the buffer appropriately (see `goto-address-fontify-p' and
 ;;;###autoload
 (define-globalized-minor-mode global-goto-address-mode
   goto-address-mode goto-addr-mode--turn-on
+  :init-value t
   :version "28.1")
 
 ;;;###autoload
diff --git a/lisp/saveplace.el b/lisp/saveplace.el
index c088facb3c..6a38ff71d1 100644
--- a/lisp/saveplace.el
+++ b/lisp/saveplace.el
@@ -151,6 +151,7 @@ disabled, i.e., the position is recorded for all files."
   "Non-nil means automatically save place in each file.
 This means when you visit a file, point goes to the last place
 where it was when you previously visited the same file."
+  :init-value t
   :global t
   :group 'save-place
   (save-place--setup-hooks save-place-mode))
diff --git a/lisp/window.el b/lisp/window.el
index a47a1216d1..01174afd64 100644
--- a/lisp/window.el
+++ b/lisp/window.el
@@ -6947,7 +6947,7 @@ that frame."
  "use a `reusable-frames' alist entry in `display-buffer-alist'."
  "24.3")
 
-(defcustom pop-up-windows t
+(defcustom pop-up-windows nil
   "Non-nil means `display-buffer' should make a new window.
 This variable is provided mainly for backward compatibility and
 should not be used in new code.  To make `display-buffer' behave
diff --git a/src/callint.c b/src/callint.c
index ce77c893f4..d14a3ea222 100644
--- a/src/callint.c
+++ b/src/callint.c
@@ -896,7 +896,7 @@ This option makes a difference in Transient Mark mode.
 When the option is non-nil, deactivation of the mark
 turns off region highlighting, but commands that use the mark
 behave as if the mark were still active.  */);
-  Vmark_even_if_inactive = Qt;
+  Vmark_even_if_inactive = Qnil;
 
   DEFVAR_LISP ("mouse-leave-buffer-hook", Vmouse_leave_buffer_hook,
 	       doc: /* Hook run when the user mouse-clicks in a window.
diff --git a/src/fns.c b/src/fns.c
index e1b3cd4321..6b81c700cc 100644
--- a/src/fns.c
+++ b/src/fns.c
@@ -6048,7 +6048,7 @@ We recommend against setting this variable non-nil, because `yes-or-no-p'
 is intended to be used when users are expected not to respond too
 quickly, but to take their time and perhaps think about the answer.
 The same variable also affects the function `read-answer'.  */);
-  use_short_answers = false;
+  use_short_answers = true;
 
   defsubr (&Sidentity);
   defsubr (&Srandom);
diff --git a/src/terminal.c b/src/terminal.c
index 52c41d4992..5f94eb6efc 100644
--- a/src/terminal.c
+++ b/src/terminal.c
@@ -650,7 +650,7 @@ syms_of_terminal (void)
   DEFVAR_LISP ("ring-bell-function", Vring_bell_function,
     doc: /* Non-nil means call this function to ring the bell.
 The function should accept no arguments.  */);
-  Vring_bell_function = Qnil;
+  Vring_bell_function = Qignored;
 
   DEFVAR_LISP ("delete-terminal-functions", Vdelete_terminal_functions,
     doc: /* Special hook run when a terminal is deleted.
