diff --git a/lisp/term/mac-win.el b/lisp/term/mac-win.el
index 20b03f3..58696a1 100644
--- a/lisp/term/mac-win.el
+++ b/lisp/term/mac-win.el
@@ -2926,8 +2926,14 @@ standard ones in `x-handle-args'."
   ;; Create the default fontset.
   (create-default-fontset)
 
-  (set-fontset-font t '(#x20000 . #x2FFFF)
-		    '("HanaMinB" . "unicode-sip") nil 'append)
+  ;; This sets up the Emoji codepoints to use prettier fonts.
+  (set-fontset-font "fontset-default" 'emoji
+                    '("Apple Color Emoji" . "iso10646-1") nil 'prepend)
+
+  ;; Set font for Chinese.
+  (dolist (charset '(kana han symbol cjk-misc bopomofo))
+    (set-fontset-font "fontset-default" charset
+                      '("PingFang SC" . "iso10646-1") nil 'prepend))
 
   ;; Create fontset specified in X resources "Fontset-N" (N is 0, 1, ...).
   (create-fontset-from-x-resource)
